app/assets/javascripts/oxalis/view/layouting/default_layout_configs.js

```diff
@@ -44,31 +60,39 @@ const SkeletonRightHandColumn = Stack(
   Panes.AbstractTreeTabView,
   Panes.Mappings,
 );
-
 const NonSkeletonRightHandColumn = Stack(Panes.DatasetInfoTabView, Panes.Mappings);
 
-const createLayout = (...content: Array<*>) => ({
-  settings: LayoutSettings,
-  dimensions: {
-    headerHeight: 18,
-    borderWidth: 1,
-  },
-  content,
-});
+const unmemoizedGetDefaultLayouts = () => {
+  let { height, width } = getGroundTruthLayoutRect();
+  // prevent default height and width
+  if (height === 500 && width === 500 && window.innerWidth) {
```

Comment:
Can we avoid these magic numbers? Two possible solutions:

- getGroundTruthLayoutRect either returns the proper height/width or null (but never the dummy values of 500). then, other callers of getGroundTruthLayoutRect would need to check for null and then use the 500 fallback value if this makes sense
- or: use window.innerWidth etc. inside getGroundTruthLayoutRect as a fallback. However, we have to be careful here, since the layout is a bit fickle during initialization. if the rect is too large, it can happen that the layout is initialized too big and that scrollbars screw up the correct dimensions. this is why, I'd go with the first option.